src/tracks/BBTrack.cpp

```diff
@@ -215,18 +215,23 @@ void BBTCOView::paintEvent( QPaintEvent * )
 {
 	QPainter p( this );
 
-	QColor col = m_bbTCO->m_useStyleColor
```

Comment:
This assignment was redundant if the next condition was satisfied. I also fixed this.